src/pfe/file-watcher/server/test/test.sh

```diff
@@ -216,8 +226,12 @@ if [[ $TEST_TYPE == "kube" ]]; then
     fi
 fi
 
-if [[ ($CLEAN_RUN == "y") ]]; then
-    cleanRun
-else
-    run
+if [ $TEST_SUITE == "functional" ]; then
+    if [[ ($CLEAN_RUN == "y") ]]; then
+        cleanRun
+    else
+        run
+    fi
+elif [ $TEST_SUITE == "unit" ]; then
+    executeTests
```

Comment:
What's the difference between `executeTests` and `run`? Do we need both?

Reply:
updated